src/bp/nlu/index.ts

```diff
@@ -82,15 +85,16 @@ export const runNluServerWithEnv = (
       const isDefined = _.negate(_.isUndefined)
       const nonNullOptions = _.pickBy(opts, isDefined)
       const options = { ...DEFAULT_STAN_OPTIONS, ...nonNullOptions }
-      const STAN_JSON_CONFIG = JSON.stringify(options)
+
+      const NLU_SERVER_CONFIG = JSON.stringify(options)
```

Comment:
The whole isDefined / negate / pick seems a bit over-complicated. What prevents you from using a simple merge ? ` const options = _.merge(DEFAULT_STAN_OPTIONS, opts)`

Reply:
tbh, I'm a little scared of the `_.merge` function as it did caused some real painfull bugs in my code in the past. 

But you're right about this being way too long to read for nothing. I could do:

```ts
const options = { ...DEFAULT_STAN_OPTIONS, ..._.pickBy(opts, x => !_.isUndefined(x)) } // which means override defaults unless value is undefined

// or even
const options = { ...DEFAULT_STAN_OPTIONS, ...opts } // as there is no real danger of overriding by undefined in this case
```

Reply:
Yeah, the last option is definitely better